introduccion-a-js-master/tareas/clase-6/task-2-class-6-scripts.js

```diff
@@ -0,0 +1,130 @@
+const $body = document.querySelector('body');
+const $buttonAddInput = document.querySelector('#add-input-button');
+const $buttonRemoveInput = document.querySelector('#remove-input-button');
+const $salaryFormContent = document.querySelector('#salary-form-content');
+const $salaryForm = document.querySelector('#salary-form');
+const $buttonCalculateSalary = document.querySelector('#calculate-salary-button');
+const $resultContent = document.querySelector('#result-values-salary');
+
+const $salaryFormTitle = document.createElement('h3');
+$salaryFormTitle.classList.add = "salary-form-title"
+$salaryFormTitle.innerText = "Enter the annual salary of each family member, only one salary per input";
+$salaryFormContent.insertBefore($salaryFormTitle, $salaryForm);
+
+$buttonAddInput.onclick = () => {
+    $salaryFormContent.style.display = "block";
+    $salaryFormTitle.style.display = "block";
+
+    const $salaryInputs = createLabelInput("Annual Salary: ", "salary-label", "salary-input");
+    $salaryForm.insertBefore($salaryInputs, $buttonCalculateSalary)
+
+    return false;
+}
+
+$buttonRemoveInput.onclick = () => {
+    const $allLabels = document.querySelectorAll('.salary-label');
+    const $allInputs = document.querySelectorAll('.salary-input');
+
+    $allLabels[$allLabels.length - 1].remove();
+    $allInputs[$allInputs.length - 1].remove();
+    $resultContent.style.display = "none";
+
+    cleanUpInput($allInputs);
+    return false;
+}
+
+$buttonCalculateSalary.onclick = () => {
+    const allWagesFromHtml = document.querySelectorAll('.salary-input');
+    const allWages = getNumbers(allWagesFromHtml);
+
+    const higherSalary = getHigher(allWages);
+    const smallerSalary = getSmaller(allWages);
+    const averageSalary = getAverage(allWages);
+    const averageMonthlySalary = getMonthlySalary(averageSalary);
+
+    $resultContent.style.display = "block";
+    const $higerSalaryElement = document.querySelector('#higher-salary');
```

Comment:
There's a typo here, ``higer`` => ``higher``

Reply:
I think the ``Element`` word is not necessary, it could be ``$higherSalaryText`` or ``$higherSalary``. What do you think?